validators/types_unix.go

```diff
@@ -40,11 +40,12 @@ var DefaultSysSpec = SysSpec{
 			{Name: "IPC_NS"},
 			{Name: "UTS_NS"},
 			{Name: "CGROUPS"},
-			{Name: "CGROUP_CPUACCT"},
+			{Name: "CGROUP_BPF"},     // cgroups v2
+			{Name: "CGROUP_CPUACCT"}, // cgroups v1 cpuacct
 			{Name: "CGROUP_DEVICE"},
 			{Name: "CGROUP_FREEZER"},
 			{Name: "CGROUP_PIDS"},
-			{Name: "CGROUP_SCHED"},
+			{Name: "CGROUP_SCHED"}, // cgroups v1 cpu
```

Comment:
It seems that Utilization Clamping depends on CONFIG_CGROUP_SCHED.
https://github.com/torvalds/linux/blob/v6.12-rc1/init/Kconfig#L1080
Since the values of Utilization Clamping can be set through the cgroup
v2 interface, CONFIG_CGROUP_SCHED must also be enabled for cgroup v2.
https://github.com/torvalds/linux/blob/v6.12-rc1/kernel/sched/core.c#L9920

Reply:
Updated.